<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheatsDialog</class>
 <widget class="QDialog" name="CheatsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cheat Menu (This feature is experimental!)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="addCheatButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>530</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Cheat</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>777</width>
        <height>521</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Cheat Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="deleteCheatButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>530</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Cheat</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>OnConfigure()</slot>
 </slots>
</ui>
